BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/gensDataPUEO.py

```diff
@@ -23,8 +23,8 @@
 # Read maxFitnessScores.csv, generationDNA.csv, and fitnessScores.csv
 
 # We use loadtxt when we need to skip over the useless text
-genDNA = np.loadtxt(g.location + "/" + str(g.GenNumber)+"_generationDNA.csv", delimiter=',', skiprows=8)
-fScores = np.loadtxt(g.location + "/" + str(g.GenNumber)+"_fitnessScores.csv", delimiter=',', skiprows=0)
+genDNA = np.loadtxt(g.location + "/" + str(g.GenNumber) + "_generationDNA.csv", delimiter=',', skiprows=9)
+fScores = np.loadtxt(g.location + "/" + str(g.GenNumber) + "_fitnessScores.csv", delimiter=',', skiprows=0)
```

Comment:
You could set `type=pathlib.Path` for the `location` above, and here use:
```suggestion
genDNA = np.loadtxt(g.location / f'{g.gen_num}_generationDNA.csv', delimiter=',', skiprows=9)
fScores = np.loadtxt(g.location / f'{g.gen_num}_fitnessScores.csv', delimiter=',', skiprows=0)
```

Reply:
Added path and fstrings to all the paths in this file in the latest commit.